docs/conf.py

```diff
@@ -47,9 +51,31 @@ def add_content(self, more_content):
         self.add_line(f"  {txt}", source_name, 2)
 
 
+class PipInstallVersionDirective(SphinxDirective):
+
+    def run(self):
+        help_text = f"pip install jwst=={version}\n"
+        paragraph_node = nodes.literal_block(text=help_text)
+        return [paragraph_node]
+
+
+class CondaInstallVersionDirective(SphinxDirective):
+
+    def run(self):
+        help_text = (
+            "conda create -n <env_name> python=3.13\n"
+            "conda activate <env_name>\n"
+            f"pip install jwst=={version}\n"
```

Comment:
I don't think this works for dev version but if we okay with the instruction broken for unreleased version, then you can mark this as resolved. Otherwise, LGTM. Thanks!

`pip install jwst==1.20.0.dev448+g44b2d9b9b`

Reply:
I think this is where we ended up - the latest docs will have erroneous instructions for installing latest released version. We can try to follow up in 12.3 dev if we want to attempt the pypi query or something similar...

Reply:
Good enough for now. We can brain storm a more elegant solution after build deadline. Thanks!